bundles/af-core/src/test/resources/form/textinput/test-content.json

```diff
@@ -4,7 +4,20 @@
     "sling:resourceType"    : "core/fd/components/form/textinput/v1/textinput",
     "name" : "abc",
     "jcr:title" : "def",
-    "fieldType": "text-input"
+    "fieldType": "text-input",
+    "customPropString": "test-custom-string",
+    "customPropStringArray": ["a", "b"],
+    "customPropBool": true,
+    "customPropBoolArray": true,
+    "customPropLong": 45,
+    "customPropLongArray": [345, 576],
+    "customPropDecimal": 45.67,
+    "customPropDecimalArray": [45.67, 90.34],
+    "customPropDate": "2022-09-13T09:00:00.000Z",
+    "customPropDateArray": ["2022-09-13T09:00:00.000Z", "2024-06-17T02:30:00.000+05:30"],
+    "fd:accidentalPrefix": "shouldNotBeInProperties",
+    "sling:accidentalPrefix": "shouldNotBeInProperties",
+    "jcr:accidentalPrefix": "shouldNotBeInProperties"
```

Comment:
@rismehta Here are the test cases for all primitive types and for excluded prefixes.
cc: @nit23uec